modal/functions.py

```diff
@@ -210,8 +210,11 @@ async def run_generator(self):
         items_received = 0
         items_total: Union[int, None] = None  # populated when self.run_function() completes
         async with aclosing(
-            _stream_function_call_data(self.client, self.function_call_id, variant="data_out")
-        ) as data_stream, aclosing(async_merge(data_stream, callable_to_agen(self.run_function))) as streamer:
+            async_merge(
```

Comment:
`async_merge` now closes its input generators so no need to wrap them in their own `aclosing` statement